state/memcached/memcached.go

```diff
@@ -102,10 +103,32 @@ func getMemcachedMetadata(metadata state.Metadata) (*memcachedMetadata, error) {
 	return &meta, nil
 }
 
+func (r *Memcached) parseTTL(req *state.SetRequest) (int, error) {
```

Comment:
I recommend changing the return type to int32 here instead to be compatible with the Memcache client